news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -0,0 +1,126 @@
+
+- If you use PrestaShop webservices for system integration, please run your integration on this beta build
+- If you build 1.7 modules or themes, please test them on this beta build
+- If you know very well some of the modified/improved/reworked Back-office pages, please give the new behaviors a try
+- If you customize some parts of the Shop, please test how your customizations behave on this new version
+- If you are hosting shops or providing maintenance services to merchants, you could import the data of one or two typical shops on a pre-production server and check the performance and the behavior of the beta build; you can also check that the update process is working as expected, depending on your favorite method
+
+For example if you are a payment module developer, just installing your module on this Beta software, processing one payment and telling us that everything is running as expected is already a great feedback !
+
+If however you find suspicious behaviors, you can
+ - [Report this as a bug on GitHub](https://github.com/PrestaShop/PrestaShop/issues) (read [how to report issues](https://devdocs.prestashop.com/1.7/contribute/contribute-reporting-issues/))
+ - Submit a bug fix by creating a [pull request](https://github.com/PrestaShop/PrestaShop/compare) (read the [contribution guidelines](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/))
+
+# Release Candidate
+
+When Beta period ends, we consider that all known regressions for this release are now registered into the Kanban. So the aim is clear: fix it all and we can ship.
+
+We fix the remaining regressions (or merge the fixes submitted by the community) and maintainers deliver a Release Candidate Build using the `1.7.7.x` branch codebase. This will be the Release Candidate 1 also known as RC1.
+
+This Build is provided to everyone, and we then start the **timer**. We wait for one week. During this week we continue testing and exploring the Build, trying to find anything that would not have been detected earlier and the community should do the same.
+
+By the end of this week, if no new regression has been reported, the RC1 becomes the final Release and **PrestaShop 1.7.7.0 is out !**
+
+... However, I dont think it ever happened than RC1 was flawless.
+
+Most of the time, some more regressions will be reported. When a new regression is reported, we fix it and we reset the timer. So there is a RC2, and if more regressions are reported, there is a RC3... until no new regressions are reported.
+
+Finally the latest built Release Candidate becomes the stable PrestaShop 1.7.7.0 software release.
+
+# Summary
+
+- The cycle begins when we start working on a minor version scope. When we complete the scope of this version, we trigger the Feature Freeze.
+
+Expected duration: 4 months
+
+- Following Feature Freeze an Alpha build is delivered and tested by QA team. QA team will then report all known defects and we will be able to plan then deliver a Beta build.
```

Comment:
```suggestion
- Following Feature Freeze, a nightly build is picked up and tested extensively by QA team, who will then report all known defects. This will allow to plan, then deliver a Beta release.
```